config/settings/aws.py

```diff
@@ -0,0 +1,22 @@
+from config.env import env
+
+DEFAULT_FILE_STORAGE = env(
```

Comment:
IMHO this should be in some place like `config/settings/file_upload.py`, rather than `aws.py`.